src/main/java/com/example/ActionsController.java

```diff
@@ -0,0 +1,41 @@
+package com.example;
+
+import com.google.actions.api.App;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.Map;
+import java.util.concurrent.ExecutionException;
+
+
+@RestController
+@RequestMapping("/actions")
+public class ActionsController {
```

Comment:
Consider renaming this to something more specific to Spring Boot, like ActionsSpringBootController.

Reply:
done.
kindly review